packages/hooks/src/useShowEffortlessLoginModal.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import useIsPasskeySupported from './useIsPasskeySupported';
+import useGetPasskeysList from './useGetPasskeysList';
+
+const useShowEffortlessLoginModal = () => {
+    const [show_effortless_modal, setShouldShowEffortlessModal] = React.useState(false);
+    const { is_passkey_supported, is_passkey_support_checking } = useIsPasskeySupported();
+    const { passkeys_list, is_passkeys_list_loading } = useGetPasskeysList();
+    const storedValue = localStorage.getItem('show_effortless_login_modal');
+    const show_effortless_login_modal = storedValue ? JSON.parse(storedValue) : false;
```

Comment:
```suggestion
    const showEffortlessLoginModal = storedValue ? JSON.parse(storedValue) : false;
```
`storedValue` in camelCase, and we are moving towards using camelCase. Should all the newly added variables here and in other files be in camelCase? Or is you team not yet moving to camelCase? :)

Reply:
@maryia-deriv , I'm using here snake_case intentionally , because this PR is not related to the new architecture. by right, we will follow common agreement while moving it to account v2